Tools/Make/Make.PeleLM

```diff
@@ -103,6 +103,8 @@ Bdirs += $(IAMR_HOME)/Source/Src_$(DIM)d
 Bdirs += $(IAMR_HOME)/Source/Godunov
 ifeq ($(USE_EB), TRUE)
    Bdirs   += $(IAMR_HOME)/Source/MOL
```

Comment:
I think this line needs to be moved outside the EB check since certain files are referenced in NavierStokesBase.cpp no matter what.

Reply:
That seems to be a weirdness on the IAMR side then. I'll try to fix that.

Reply:
Hum ... MOL isn't split between EB and non-EB so there might not be a way around it.

Reply:
My code worked fine when I moved it out of the if statement.

Reply:
Indeed, they are now using non-EB MOL in NSB.